docs/pages/setup/security.mdx

```diff
@@ -7,5 +7,6 @@ layout: tocless-doc
 <ul>
   <li>
     [Reducing the blast radius of attacks](./security/reduce-blast-radius.mdx). Prevent attackers from accessing your infrastructure even if they manage to obtain passwords or certificates.
+    [Hardened container images](./security/hardened-container-images.mdx). How to run Teleport in hardened container images.
```

Comment:
```suggestion
</li>
<li>
[Hardened container images](./security/hardened-container-images.mdx). How to run a Teleport container in heavily regulated environments.
</li>
```
- Added the second link as a list item, rather than in the same list item as "Reducing the blast radius"
- Suggested changing the description so it brings out the benefit of using hardened container images